2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -103,6 +119,7 @@ collector {
     }
 
     # Incoming events are stored in a buffer before being sent to Kinesis/Kafka.
+    # Note: Buffering is not supported by NSQ; will be ignored    
```

Comment:
Sorry, since I did not see the direct usage of these field, I thought they are not used. I will make necessary changes then.